core/src/bundle/usBundleAbstractTracked_p.h

```diff
@@ -281,6 +278,8 @@ class BundleAbstractTracked : public MultiThreaded<MutexLockingStrategy<>,WaitCo
    */
   std::atomic<int> trackingCount;
 
+  BundleContext* bc;
```

Comment:
Could be `BundleContext* const` instead. If the `GetLogSink()` method is moved to the bunde private classe, this would become `BundleContextPrivate * const`.